frontend/wallet/src/render/Page.re

```diff
@@ -91,7 +84,7 @@ let make = (~message) => {
   let testButton = (str, ~f) => {
```

Comment:
What was the plan to kill this file again?

Reply:
Just moving the logic we need over to `ReactApp`, there's some like boilerplate and unused vars and stuff so I'm just cleaning it up by moving stuff into a new file.